### DISCUSSION TITLE
Computer Modern Monospace

### DISCUSSION THREAD

**Discussion by johnmapeson** [typst/typst#4928] [Q&A] [answered]
It seems Typst comes with New Computer Modern font:

```
#set text(font: "New Computer Modern")
#lorem(10)

Works fine
```

But whereas <https://ctan.org/pkg/newcomputermodern> implicitly says there is a monospace version, `typst fonts` doesn't mention neither New Computer Modern Monospace, nor New Computer Modern Monospaced. And therefore neither of the following commands work:

- `#set text(font: "New Computer Modern Monospace")`
- `#set text(font: "New Computer Modern Monospaced")`

Is there a simple way to make New Computer Modern Monospace[d] work with Typst? (Or at least which monospaced Unicode font that is supplied with Typst would you suggest as a companion to New Computer Modern?)

**Comment by PgBiel** [comment, marked as answer]
New Computer Modern's monospace version isn't packaged with Typst by default (see [list](https://github.com/typst/typst-assets/tree/main/files/fonts)). You can download the OTF file here: https://ctan.org/texarchive/fonts/newcomputermodern/otf

And then attach it to your web app project, or (using the CLI) write `--font-path folder_with_the_otf_file`.
